products/proxmox-ve.md

```diff
@@ -21,8 +21,8 @@ releases:
 -   releaseCycle: "7"
     releaseDate: 2021-07-06
     eol: false
-    latest: "7.3"
-    latestReleaseDate: 2022-11-22
+    latest: "7.4"
```

Comment:
We could use `__LATEST__` instead of `__RELEASE_CYCLE__` in the `changelogTemplate`.

Reply:
#2742